scripts/generateAutocompleteGraph.ts

```diff
@@ -191,51 +194,67 @@ fetch('https://catfact.ninja/fact', { method: 'GET' })
       return addSearchQueryCharacter(newNode, characters.slice(1), data);
     }
 
+    function addWithParents(
```

Comment:
It would probably be worth adding a short (even few words) comment for what this method does.

Reply:
Great idea, added in latest commit